app/services/export/s3_uploader_config.rb

```diff
@@ -19,10 +23,18 @@ def secret_key
     def credentials
       JSON.parse(ENV.fetch("VCAP_SERVICES"))
         .fetch("aws-s3-bucket")
-        .find { |config| config.fetch("name").match?(/s3-export-download-bucket/) }
+        .find { |config| config.fetch("name").match?(bucket_name_regex) }
         .fetch("credentials")
     rescue KeyError, NoMethodError => _error
       raise "AWS S3 credentials not found"
     end
+
+    private
+
+    attr_reader :public
+
+    def bucket_name_regex
+      public ? /s3-export-download-bucket$/ : /s3-export-download-bucket-private/
```

Comment:
I was wondering that too, it's not listed [here](https://docs.ruby-lang.org/en/2.2.0/keywords_rdoc.html) but neither is `private`. I think I'll change this as even though it's working as expected, it might be confusing, and the syntax highlighting here does make it look reserved...

Reply:
Have changed this to `use_public_bucket` (it's not the best name, but we'll be getting rid of it soon, once we move spending breakdown over)